services/search/pkg/search/service_test.go

```diff
@@ -345,6 +352,11 @@ var _ = Describe("Searchprovider", func() {
 				Expect(match.Entity.Name).To(Equal("Shared.pdf"))
 				Expect(match.Entity.Ref.ResourceId.OpaqueId).To(Equal(mountpointSpace.Root.OpaqueId))
 				Expect(match.Entity.Ref.Path).To(Equal("./to/Shared.pdf"))
+				Expect(match.Entity.RemoteItemId).To(Equal(&searchmsg.ResourceID{
+					StorageId: grantSpace.RootInfo.Id.StorageId,
```

Comment:
Please always use Getters from the protobuf generated types `GetRootInfo().GetID().GetStorageID()`

Reply:
But this is a test and `grantSpace` is a mock, so `GetRootInfo().GetID().GetStorageID()` seems to be unavailable on this mock. Do I need to add something to make this work? Other tests also don't use getters though 🤔

Reply:
That was just my „inner linter“ parsing the diff. Not an issue in tests.